mne/source_space/tests/test_source_space.py

```diff
@@ -679,6 +679,7 @@ def test_source_space_from_label(tmp_path, pass_ids):
     _compare_source_spaces(src, src_from_file, mode="approx")
 
 
+@pytest.mark.slowtest
 @testing.requires_testing_data
 def test_source_space_exclusive_complete(src_volume_labels):
```

Comment:
AFAICT this is the only test function that uses `src_volume_labels`, and since it's already marked with `requires_testing_data` I only copied over the `slowtest` mark.